app/javascript/app/components/country-ghg-emissions/country-ghg-emissions.js

```diff
@@ -63,9 +73,17 @@ class CountryGhgEmissionsContainer extends PureComponent {
   }
 
   handleSourceChange = category => {
-    this.updateUrlParam({ name: 'source', value: category.value }, true);
+    if (category) {
+      this.updateUrlParam({ name: 'source', value: category.value }, true);
+    }
   };
 
+  handleYearHover = throttle(data => {
+    if (data) {
+      this.updateUrlParam({ name: 'year', value: data.activeLabel });
```

Comment:
I know this works nicely, but is it needed in the URL as it is bound to the tooltip which will only show on hover? Would it be better in the state? Not sure this param needs to be shared?